### PR TITLE
polish up sidebar

### PR DESCRIPTION
**Pull request by Omarzintan**

### CODE REVIEW

File: src/main/webapp/mapScript.js [hunk 1]
```diff
@@ -141,6 +141,8 @@ function highlightNearbyLocation(map, placeType, currentLocation) {
       for (var i = 0; i < results.length; i++) {
         createMarker(map, results[i].geometry.location, results[i].place_id);
       }
+      message.innerText = results.length - 1 + ' location(s) found.';
```
Comment (rglassett) [inline src/main/webapp/mapScript.js]: Is there a reason to subtract 1 here? An array of length 5 will contain 5 results/locations.

File: src/main/webapp/mapScript.js [hunk 2]
```diff
@@ -273,7 +278,10 @@ function displayPlaceInfo(place, placeId) {
   nameElement = document.createElement('h2');
   ratingElement = document.createElement('span');
   ratingElement.id = 'stars';
-  addressElement = document.createElement('p');
+  addressDiv = document.createElement('div');
+  addressIcon = document.createElement('img');
+  addressIcon.src = '/images/black-marker.png';
+  addressElement = document.createElement('h5');
```
Comment (rglassett) [inline src/main/webapp/mapScript.js]: Nit: Rename to `addressHeading`

**Review by rglassett — APPROVED** [review]

File: src/main/webapp/mapScript.js (hunk 1 above)
Comment (Omarzintan) [reply src/main/webapp/mapScript.js]: Oh, thanks. I will correct that

File: src/main/webapp/mapScript.js (hunk 2 above)
Comment (Omarzintan) [reply src/main/webapp/mapScript.js]: will do!

**Pull request merged**